docs/faq.md

```diff
@@ -137,3 +137,7 @@ the [Architecture documentation](./architecture.md#docker-networking)
 The user interface is based in the open-sourced [Apache Superset](http://superset.apache.org),
 so you can also refer to [Superset tutorials](http://superset.apache.org/tutorial.html)
 for advanced usage of the web interface.
+
+## I Get IOError: [Errno 13] Permission denied: u'./.env'
+
+This may happen if you have installed Docker from a snap package. This installation mode is not supported, please install it following [the official documentation](https://docs.docker.com/install/) (See [#78](https://github.com/src-d/sourced-ce/issues/78)).
```

Comment:
What if linking our docs about how to install docker? We already provide some links to the different OS.